docs/docs/features/encoders.md

```diff
@@ -23,25 +23,37 @@ Keyboards and macropads with encoder support will typically take the two EC11 pi
 
 ### Rotation
 
-Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`, which is defined in each keymap layer in the following format:
+Rotation is handled separately as a type of sensor. The behavior for this is set in `sensor-bindings`. See [Sensor Rotation](../behaviors/sensor-rotate.md) for customizing this behavior.
 
-```
-sensor-bindings = <BINDING CW_KEY CCW_KEY>;
-```
-
-- `BINDING`, for now, has only one behavior available; `&inc_dec_kp` for key presses (see [Key Press](../behaviors/key-press.md) for details on available keycodes).
-- `CW_KEY` is the keycode activated by a clockwise turn.
-- `CCW_KEY` is the keycode activated by a counter-clockwise turn.
-
-Additional encoders can be configured by adding more `BINDING CW_KEY CCW_KEY` sets immediately after the first.
+Additional encoders can be configured by adding more bindings immediately after the first.
 
 As an example, a complete `sensor-bindings` for a Kyria with two encoders could look like:
 
 ```
-sensor-bindings = <&inc_dec_kp C_VOL_UP C_VOL_DN &inc_dec_kp PG_UP PG_DN>;
+/ {
+    behaviors {
+        rgb_encoder: rgb_encoder {
+            compatible = "zmk,behavior-sensor-rotate";
+            label = "RGB_ENCODER";
+            #sensor-binding-cells = <0>;
+            bindings = <&rgb_ug RGB_BRD>, <&rgb_ug RGB_BRI>;
+        };
+        arrow_encoder: arrow_encoder {
+            compatible = "zmk,behavior-sensor-rotate";
+            label = "ARROW_ENCODER";
+            #sensor-binding-cells = <0>;
+            bindings = <&kp LEFT>, <&kp RIGHT>;
+        };
+    };
+
+    keymap {
+        ...
+        sensor-bindings = <&rgb_encoder &arrow_encoder>;
+    };
+};
 ```
 
-Here, the left encoder is configured to control volume up and down while the right encoder sends either Page Up or Page Down.
+Here, the left encoder is configured to control rgb brightness while the right encoder sends either Left or Right.
```

Comment:
Similar note as that for `sensor-rotate.md` for capitalization, but also enhancing clarity on the behaviour of the `arrow_encoder` binding.

```suggestion
Here, the left encoder is configured to control RGB brightness while the right encoder sends either the left or right arrow keys.
```